### ISSUE TITLE
reference gnu screen with example how to use 3d-ascii-viewer as screensaver?

### ISSUE THREAD

**Issue by alexmyczko** [autopawn/3d-ascii-viewer#14]
https://www.gnu.org/software/screen/manual/html_node/Screen-Saver.html

**Comment by autopawn** [comment]
In my opinion, this probably is more deserving of a tutorial in the wiki than to be in the README.

Is `screen` commonly used? I had to install it in my system (Pop!_OS).

By adding:
```
blankerprg /home/fcasas/3d-ascii-viewer-c/3d-ascii-viewer --color /home/fcasas/3d-ascii-viewer-c/models/fox.obj
idle 5
```
to my `screen.rc`, I got it working after starting `screen` and waiting for 5 seconds, but a problem is that it doesn't revert the color palette after the program it is closed, when using the `--color` mode.
